### PR TITLE
Change inappropriate usage of meanwhile vs while

### PR DESCRIPTION
**Pull request by reinsch82**
### Reviewing cheatsheet

Before merging the PR, comments starting with 
- ❌ ❓**must** be fixed
- 🤔 💅 **should** be fixed
- 💭 **may** be fixed
- 🎉 celebrate happy things

### CODE REVIEW

**Comment by reinsch82** [comment]
@bot-gradle test this

**Review by donat — APPROVED** [review]

**Comment by reinsch82** [comment]
@bot-gradle merge

**Review by ghale — APPROVED** [review]
LGTM

File: platforms/ide/problems/src/main/java/org/gradle/problems/internal/transformers/PluginIdLocationTransformer.java [hunk 1]
```diff
@@ -46,7 +46,7 @@ public Problem transform(Problem problem) {
                         problem.getLocations().add(new PluginIdLocation(pluginId));
                     }
                 } catch (Exception ex) {
-                    throw new GradleException("Problem meanwhile reporting problem", ex);
+                    throw new GradleException("Problem while reporting problem", ex);
```
Comment (ghale) [inline platforms/ide/problems/src/main/java/org/gradle/problems/internal/transformers/PluginIdLocationTransformer.java:49]: The use of `while` here is correct, but I would argue that this error message could be better.  I'm not sure what might make sense to include, but "Problem while reporting problem" seems overly vague.  For example, including identifying information from the `Problem` or the `OperationDetails` might make this quite a bit better.

File: subprojects/precondition-tester/src/test/groovy/org/gradle/test/precondition/PreconditionProbingTest.groovy [hunk 2]
```diff
@@ -32,7 +31,7 @@ import org.junit.jupiter.api.Assumptions
  *         This is normal, and simply signifies that the precondition is not satisfied on the current system
  *     </li>
  *     <li>
- *         <b>Failed:</b> there was an error meanwhile checking the combination.
+ *         <b>Failed:</b> there was an error while checking the combination.
```
Comment (tresat) [inline subprojects/precondition-tester/src/test/groovy/org/gradle/test/precondition/PreconditionProbingTest.groovy:34]: 💭 Why are there so many different message prefixes when reporting an error:

- Problem while ...
- An error happened while ...
- Failed: there was an error while ...

Could these be standardized?

Comment (reinsch82) [reply subprojects/precondition-tester/src/test/groovy/org/gradle/test/precondition/PreconditionProbingTest.groovy:34]: Good point, my pet peeve (as I just learned not "beef" :D ) is not yet big enough with this.

**Pull request merged**